src/tools/miri/src/shims/x86/mod.rs

```diff
@@ -69,7 +70,7 @@ pub(super) trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                 if is_u64 && this.tcx.sess.target.arch != "x86_64" {
                     return interp_ok(EmulateItemResult::NotSupported);
                 }
-
+                // TODO: How do we handle ExternAbi::Unadjusted??
                 let [c_in, a, b, out] =
                     this.check_shim(abi, ExternAbi::Unadjusted, link_name, args)?;
```

Comment:
@RalfJung There are quite a few places use ``ExternAbi::Unadjusted``. I wonder why this is used instead of ``ExternAbi::C { unwind: false }``? 

Also, is there anything in https://doc.rust-lang.org/nightly/nightly-rustc/rustc_target/callconv/enum.Conv.html that can replace ``ExternAbi::Unadjusted``?

Reply:
The SIMD shims use `Unadjusted` because that's what is necessary to make them work. :)

According to [this](https://github.com/rust-lang/rust/blob/948cec0fad6d87f7f5993517f913aa0454b78674/compiler/rustc_ty_utils/src/abi.rs#L314), `ExternAbi::Unadjusted` maps to `Conv::C`, so please use that.

Reply:
> According to [this](https://github.com/rust-lang/rust/blob/948cec0fad6d87f7f5993517f913aa0454b78674/compiler/rustc_ty_utils/src/abi.rs#L314), ExternAbi::Unadjusted maps to Conv::C, so please use that.

The mapping is interesting... thanks for pointing it out!